llvm/unittests/DebugInfo/LogicalView/CMakeLists.txt

```diff
@@ -1,17 +1,22 @@
 set(LLVM_LINK_COMPONENTS
+  ${LLVM_TARGETS_TO_BUILD}
```

Comment:
Doesn't this make the next 3 lines superfluous?

Reply:
@nico That is a good point. With your suggested change, all pre-commit test pass.
Opened https://github.com/llvm/llvm-project/pull/118052